conf/application-json-logger.xml

```diff
@@ -1,14 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<configuration>
-
-    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
-        <encoder class="uk.gov.hmrc.play.logging.JsonEncoder"/>
-    </appender>
-
-    <logger name="application.uk.gov.hmrc.forexrates.services.EcbForexServiceImpl" level="INFO"/>
-    <logger name="application.uk.gov.hmrc.forexrates.services.ExchangeRateLogger" level="INFO"/>
```

Comment:
Same as [here](https://github.com/hmrc/one-stop-shop-forex-rates/pull/30#discussion_r1801102188)

Reply:
Added Logs to app-configs